requirements/contraints.txt

```diff
@@ -0,0 +1,3 @@
+# required by aiohttp 3.6.3
+multidict<5.0
+yarl<1.6.0
```

Comment:
Is it required for us to have `constraints.txt`? if by having it we solve some problem, then let's do it in a separate issue + PR to have more control on this

Reply:
aiohttp 3.6.3 which we use doesn't support yarl 1.6 and multidict 5.0, so I had to add these constraints. Okay, I'll create a separate PR